docs/source/documentation/user-management.erb

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
This file needs to be named `user-management.html.md.erb` in order for it to be rendered as expected.